pkg/repo/mysql/message.go

```diff
@@ -0,0 +1,150 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+func (r *Repo) SaveMessages(ctx context.Context, serviceID uuid.UUID, messages *model.Messages) error {
+	_, err := r.LoadService(ctx, serviceID)
+	if err != nil {
+		return fmt.Errorf("repo: load service: %w", err)
+	}
+
+	tx, err := r.db.BeginTx(ctx, nil)
+	if err != nil {
+		return fmt.Errorf("repo: begin tx: %w", err)
```

Comment:
I suggest the following to have some context to the intent

```suggestion
		return fmt.Errorf("repo: begin tx to save messages: %w", err)
```